templates/2024/navbar.html

```diff
@@ -49,15 +50,53 @@
                   <a class="nav-link g-px-0 g-color-black" href="/2024/faqs/">FAQs</a>
                 </li> 
                 -->
-                <li class="dropdown-item">
+                <!-- <li class="dropdown-item">
```

Comment:
We could as well delete this(the comment plus the list items) as is dead code since is catered for under Attendee Guide